src/main/java/world_cities/ParseWorldCities.java

```diff
@@ -0,0 +1,59 @@
+package world_cities;
+
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVRecord;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ParseWorldCities {
+    private double lon;
+    private double lat;
+    private List<City> cities;
+
+    public ParseWorldCities(double lon, double lat) throws IOException {
```

Comment:
Why are you passing lon and lat?

Reply:
So that it can know what the lat and lon from the Servlet are to find the closest city? Did I do this totally wrong?

Reply:
Actually just removed it and my unit tests pass, so I dont think I needed it